docker/omlbotlookup/data_access.R

```diff
@@ -23,22 +23,23 @@ con <- dbConnect(MySQL(), user = mysql_username, password = mysql_password, dbna
 # See /docker/preparation/prepare_parameter_ranges.R for instructions.
 # This file contains parameter range data obtained from the omlbot-sourcecode.
 parameter_ranges = readRDS("parameter_ranges.Rds")
+names(parameter_ranges) = paste0("mlr.", names(parameter_ranges))
 
 
 #' Return a list of parameter definitions. This list contains every necessary parameter for the given algorithm.
 #'
 #' @param task_id This is `task_id` from the table `run` in the database.
 #'
 #' @return A named list with one entry for each needed parameter.
-get_params_for_algo = function(algo_name) {
+get_params_for_algo = function(algo_id) {
```

Comment:
algo_id ist eigentlich ungleich algo_name. Ein algo_name entspricht ja mehreren algo_ids.

Reply:
Ok du willst also eigentlich die option entfernen, nach einzelnen algo_ids zu fragen. find ich in ordnung, allerdings hätte ich da den namen algo_name bevorzugt. ist aber ja im endeffekt egal.